module/advancement/types/scale-value.mjs

```diff
@@ -69,7 +259,7 @@ export class ScaleValueAdvancement extends Advancement {
 
   /** @inheritdoc */
   titleForLevel(level, { configMode=false }={}) {
-    const value = this.formatValue(level);
+    const value = this.valueForLevel(level)?.formatted;
```

Comment:
Since we have `toString` defined, I think this can just be:

```suggestion
    const value = this.valueForLevel(level);
```

Reply:
`toString` returns the formula value, not the display value (this way when it is in a roll formula you can just do `@scale.monk.martial-arts` to get the proper formula version and not the display version, which might not work in formulas).